src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-form/rgw-bucket-form.component.html

```diff
@@ -277,6 +296,24 @@
                     i18n>Retention Days must be a positive integer.</span>
             </div>
           </div>
+          <!-- Alerts -->
+          <div class="form-group row">
+            <div class="cd-col-form-label"></div>
+            <div class="cd-col-form-input">
+              <cd-alert-panel
+                type="info"
+                *ngIf="bucketForm.getValue('lock_enabled')"
+                class="me-1"
+                i18n-title>
+                  Bucket Versioning can't be disabled when Object Locking is enabled.
+              </cd-alert-panel>
+              <cd-alert-panel
+                type="warning"
+                *ngIf="bucketForm.getValue('lock_enabled')">
```

Comment:
If I'm not mistaken, `alert-panel`s should go on top of the relevant info

Reply:
Can you elaborate:
* after object locking legend ?
* after check box ?

Also, I kept this way as it looked clean to me, but yes I missed to check what is correct UXD pattern.
Right now we are following carbon so I need to check that probably.

Thanks

Reply:
For this very case, after object locking and on before the input fields, we don't really have much `cd-alert-panel` on forms, so not so sure.
But as far as I know, alerts panel usually go on top of the content that inform about

Reply:
Since we have different sections here so if we place right at the bottom it would be too late for user to see and then scroll back.
So, placing at the bottom of section seems fine , wdyt ?